mtprotoproxy.py

```diff
@@ -119,10 +120,22 @@ def decrypt(self, data):
 KEY_LEN = 32
 IV_LEN = 16
 HANDSHAKE_LEN = 64
-MAGIC_VAL_POS = 56
+PROTO_TAG_POS = 56
 DC_IDX_POS = 60
 
-MAGIC_VAL_TO_CHECK = b'\xef\xef\xef\xef'
+PROTO_TAG_ABRIDGED = b'\xef\xef\xef\xef'
+PROTO_TAG_INTERMEDIATE = b'\xee\xee\xee\xee'
+
+class RpcFlags(enum.Flag):
```

Comment:
This is 3.6+ only, there's a backport, but it can also be refactored to just use constants